sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -775,6 +775,7 @@ static struct snd_soc_dai_link *sof_card_dai_links_create(struct device *dev,
 
 		} else {
 			max_98357a_dai_link(&links[id]);
+			links[id].dpcm_capture = 1;
```

Comment:
the maxim 98357a does not provide a capture dai, 

````c
static struct snd_soc_dai_driver max98357a_dai_driver = {
	.name = "HiFi",
	.playback = {
		.stream_name	= "HiFi Playback",
		.formats	= SNDRV_PCM_FMTBIT_S16 |
					SNDRV_PCM_FMTBIT_S24 |
					SNDRV_PCM_FMTBIT_S32,
		.rates		= SNDRV_PCM_RATE_8000 |
					SNDRV_PCM_RATE_16000 |
					SNDRV_PCM_RATE_32000 |
					SNDRV_PCM_RATE_44100 |
					SNDRV_PCM_RATE_48000 |
					SNDRV_PCM_RATE_88200 |
					SNDRV_PCM_RATE_96000,
		.rate_min	= 8000,
		.rate_max	= 96000,
		.channels_min	= 1,
		.channels_max	= 2,
	},
	.ops    = &max98357a_dai_ops,
};

````
so that change seems invalid.

What makes you think this is relevant?